integrations.json

```diff
@@ -137,6 +137,64 @@
       }
     ]
   },
+  {
+    "name": "NServiceBus",
+    "type": "Trace",
+    "method_replacements": [
+      {
+        "caller": {},
+        "target": {
+          "assembly": "NServiceBus.Core",
+          "type": "NServiceBus.EndpointConfiguration",
+          "method": ".ctor",
+          "signature_types": [
+            "System.Void",
+            "System.String"
+          ],
+          "minimum_major": 8,
+          "minimum_minor": 0,
+          "minimum_patch": 0,
+          "maximum_major": 8,
+          "maximum_minor": 65535,
+          "maximum_patch": 65535
+        },
+        "wrapper": {
+          "assembly": "OpenTelemetry.AutoInstrumentation",
+          "type": "OpenTelemetry.AutoInstrumentation.Instrumentations.NServiceBus.EndpointConfigurationIntegration",
+          "action": "CallTargetModification"
```

Comment:
side note: do we want to support any other type of action? AFAIK this is the only one we support. See https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/345beca02a83639c82487828eec59c8100c9afdf/src/OpenTelemetry.AutoInstrumentation.Native/integration_loader.cpp#L203

Reply:
We can remove it from the code if we not treat integration.json file as a public contract.
Otherwise I would preferer to keep it as it is.

Reply:
I would say it is "public" in the same lecel as our plugins API.

Reply:
I created https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1883